components/dashboard/donation/removeDonationModalBody.js

```diff
@@ -8,13 +9,14 @@ import { deleteDonation } from '../../../client'
 
 const RemoveDonation = ({ updateDonorStatus, onClose }) => {
   const [isDeleting, setIsDeleting] = useState(false)
+  const router = useRouter()
   const [error, setError] = useState('')
 
   const handleRemoveDonation = async () => {
     setIsDeleting(true)
     try {
-      const response = await deleteDonation()
-
+      const organizationId = router.query.organizationId
+      const response = await deleteDonation({ organizationId })
```

Comment:
what is the experience when org id is undefined?

Reply:
well.... the url has the org id in it so not a gr8 experience if no org id